app/scripts/controllers/network/createJsonRpcClient.test.js

```diff
@@ -0,0 +1,194 @@
+/**
```

Comment:
same tests suite as createInfuraClient, just moving it closer towards using the openrpc document referenced in the comment below

Reply:
I will add that back. 

At some point I'd like to revist these tests and use a combination of the ethereum/execution-apis + metamask/api-specs openrpc document